app/javascript/packs/routes/AssetManagerRoutes.jsx

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
-import {
-  Route,
-  Switch
-} from 'react-router-dom';
-
 import PortfolioContainer from '../containers/PortfolioContainer';
 import BuildingContainer from '../containers/BuildingContainer';
 
+import { Route, Switch, Redirect } from 'react-router-dom';
+
 class AssetManagerRoutes extends React.Component {
   render() {
     return (
       <div>
         <Switch>
           <Route path="/portfolios/:pId" component={PortfolioContainer} />
+
+          {/* If no questionnaire mode is specified, the default should be /edit */}
+          <Route exact path="/buildings/:bId" render={({match}) => (<Redirect to={`${match.url}/edit`} />)} />
```

Comment:
The questionnaire view redundancy across `AssetManagerRoutes` and `BuildingOperatorRoutes` has been fine so far, but- thoughts on moving it into a `QuestionnaireRoutes` component to be shared? Especially if we end up including all of the mode routes in here too, it may be helpful to have them all in one file and tossed into here.

Reply:
Discussed in meeting -- `BuildingContainer` is basically playing the role of `QuestionnaireRoutes` ;D I renamed it for clarity